docs/source/developer/test_suite.rst

```diff
@@ -4,32 +4,80 @@ Test Suite
 .. contents:: Contents
   :local:
 
+
+Running the Test Suite
+==============================================================================
+
+``./csu dev test`` will run the entire test suite. For a full list of test commands available, see :doc:`../getting_started/helper_commands`.
```

Comment:
Might be worth changing 'a full list of test commands available' to 'running a specific test suite, viewing code coverage, and more testing commands'.